### DISCUSSION TITLE
Home Assistant OS 13.2

### DISCUSSION THREAD

**Discussion by sairon** [home-assistant/operating-system#3626] [Announcements]
## Home Assistant Operating System

* Fix guest-agent not starting with HyperV Enlightenments enabled (#3592) @sairon
* Bump Buildroot to 2024.02.6 (#3586) @sairon
* Relocate HAOS Systemd drop-ins to /usr/lib/systemd (#3582) @sairon
* Bump Docker to v27.2.0 (#3576) @sairon
* Bump Buildroot to 2024.02.5 (#3550) @sairon

## Home Assistant Green

* Linux: Update kernel to 6.6.54 (#3614) @sairon

## Open Virtual Appliance

* Fix guest-agent not starting with HyperV Enlightenments enabled (#3592) @sairon
* Linux: Update kernel to 6.6.54 (#3614) @sairon
* Enable debug logging for GRUB rescue shell options (#3581) @sairon
* Fix missing Intel AX210 firmware files, remove unused firmwares (#3556) @sairon

## Generic x86-64

* Use GRUB2 legacy loader for Macbook Air A1370 (#3593) @sairon
* Linux: Update kernel to 6.6.54 (#3614) @sairon
* Enable debug logging for GRUB rescue shell options (#3581) @sairon
* Use GRUB legacy loader for HP t520 to fix boot with GRUB 2.12 (#3562) @sairon
* Fix missing Intel AX210 firmware files, remove unused firmwares (#3556) @sairon

## Hardkernel ODROID

* Linux: Update kernel to 6.6.54 (#3614) @sairon

## ASUS Tinker

* Linux: Update kernel to 6.6.54 (#3614) @sairon

## Khadas VIM Series

* Linux: Update kernel to 6.6.54 (#3614) @sairon

## Generic aarch64

* Linux: Update kernel to 6.6.54 (#3614) @sairon
* Enable debug logging for GRUB rescue shell options (#3581) @sairon
* Fix missing Intel AX210 firmware files, remove unused firmwares (#3556) @sairon

## Build System

* Add test for no failed Systemd units (#3583) @sairon
* Tests: update labgrid to v24.0.1 (#3584) @sairon
* Purge Cloudflare cache for all files after uploading R2 artifacts (#3568) @sairon

## Dependencies

* Bump docker/setup-buildx-action from 3.6.1 to 3.7.1 (#3613) @dependabot
* Bump docker/build-push-action from 6.7.0 to 6.9.0 (#3604) @dependabot

## Other Changes

* Update buildroot branch in .gitmodules to 2024.02.x-haos (#3589) @adeepn
* Add OHF logo to README (#3603) @mrdarrengriffin

<hr /><em>This discussion was created from the release <a href='https://github.com/home-assistant/operating-system/releases/tag/13.2'>Home Assistant OS 13.2</a>.</em>

**Comment by m-rossi** [comment]
Someone with RaspberryPi 5 and ZHA already updated and can check if https://github.com/home-assistant/operating-system/issues/3352 is still an issue?

**Reply by agners** [reply]
There is no upgrade to the Raspberry Pi kernel or another change which might address this particular issue. So I don't expect that this release fixes that issue.

**Comment by Anto79-ops** [comment]
There are no issues on my NUC 12 (x86-64, bare-metal) with USB SkyConnect and USB Aeotec Z-Stick 7.

System is running normally.

**Comment by jbcodemonkey** [comment]
Been using this platform for seven years, from a time when all of the configuration was editing yaml. I still have one dashboard that preserves what it looked like in the beginning as a reminder of how far this has come.

Now it's a self-maintaining OS that lives in a VM. I just click 'update', and it goes. I do dashboard updates on my phone in seconds, and the new tiles and headings are wonderful new ways to visualize and interact.

All of you are doing excellent work, and I am grateful every day. :)

**Comment by Tanjohnson99** [comment]
Hello, Since the update, the dongle "SkyConnect" are not working.

**Comment by maurolemos** [comment]
Was anything done to fix the issues multiple folks have reported with 13.1 boot looping on RPI3s and RPI4s?
https://community.home-assistant.io/t/haos-13-1-upgrade-unable-to-boot/762530

**Comment by Jaber7** [comment]
Please fix Gree AC integration not update automatically
